hippiehug-package/hippiehug/Chain.py

```diff
@@ -14,6 +14,17 @@ def check_hash(key, val):
         raise Exception("Value has the wrong hash.")
 
 
+def sort_dicts(unsorted):
+    if isinstance(unsorted, dict):
+        return sorted({k: sort_dicts(v) for k,v in unsorted.items()})
```

Comment:
Particularly, this expression drops the values of the dict:
```python
In [1]: sorted({'a': 1, 'b': 2})
Out[1]: ['a', 'b']
```

Reply:
Good catch!